### DISCUSSION TITLE
Whether NUTRIENT_PATH2 needs to be deleted in the calculation of the NHANES 1999-2002 Dietary pattern index?

### DISCUSSION THREAD

**Discussion by Whitney678** [jamesjiadazhan/dietaryindex#311] [Q&A] [answered]
Hi Author, Thanks for the R package, I would like to ask you a question. When I calculate NHANCES 1999-2002 Dietary pattern index, for example HEI2015 for 2001-2002,do I need to delete NUTRIENT_PATH2 = NHANES_20012002$NUTRIENT2, NUTRIENT_IND_PATH2 = NHANES_20012002$ NUTRIENT_IND2)

**Comment by jamesjiadazhan** [comment, marked as answer]
As NHANES 2001-2002 does not have the day 2 data, those parameters for day 2 are not necessary. Just delete them.
